scripts/pgs/LDpred2/ldpred2.R

```diff
@@ -233,7 +234,11 @@ drops <- c("_NUM_ID_.ss", "rsid.ss", 'block_id', 'pos_hg18', 'pos_hg38')
 df_beta <- df_beta[ , !(names(df_beta) %in% drops)]  
 
 cat('\n### Loading LD reference from ', fileLD, '\n')
-tmp <- tempfile(tmpdir = "tmp-data")
+if (file.exists(parsed$tmp_dir)) {
+  tmp_file <- tempfile(tmpdir=parsed$tmp_dir)
```

Comment:
Similar to my comment on the prior PR, consider doing these tests (file.exists(...)) before reading in any data so that the script stops early on. It's also good in for the testing environment if you wish to test this error as it reduces time and memory consumption.